detox/src/devices/Device.js

```diff
@@ -235,7 +235,7 @@ class Device {
   }
 
   async resetContentAndSettings() {
-    await this.deviceDriver.resetContentAndSettings(this._deviceId);
+    await this.deviceDriver.resetContentAndSettings(this._deviceId, this._bundleId);
```

Comment:
@LeoNatan, I now wonder whether `resetContentAndSettings` process can be made app-specific, not global for the whole device (simulator)?

Reply:
Not possible. On iOS there is no such functionality, and it is the same with simulators. We can delete the app, but that's not the same thing and doesn't reset the settings related to the app.